toko/templates/products/detail.html

```diff
@@ -1,43 +1,50 @@
 {% extends 'base.html' %}
 {% block content %}
+<div class="container-fluid">
+<div class="content-wrapper">
+  <div class="item-container">
+    <div class="container">
+      <div class="col-md-8">
+        <h2>{{ product }} </h2>
+        {% for photo in product.photos.all %}
```

Comment:
Kalo fieldnya many-to-many atau many-to-one kita pake `prefetch_related`, sedangkan kalo one-to-many kita pake `select_related`

https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.prefetch_related